src/nvim/fileio.c

```diff
@@ -5352,10 +5371,12 @@ static int au_find_group(char_u *name)
   return AUGROUP_ERROR;
 }
 
-/*
- * Return TRUE if augroup "name" exists.
- */
-int au_has_group(char_u *name)
+/// Return true if augroup "name" exists.
+///
+/// @param name The augroup name
+///
+/// @return true if augroup "name" exists.
```

Comment:
This is redundant. I don't know what is the best approach but I recall reading an issue about this, can't find it though.